### DISCUSSION TITLE
Touch Screen and  SD support

### DISCUSSION THREAD

**Discussion by TommieEdwardBerry** [community/community#120730] [Programming Help]
### Select Topic Area

Question

### Body

Will the touch screen and  SD support be added in the future? 
I am using the 4 inch TFT with touch and SD.
I use ESP32-S3-WROOM-1 (MCN16R8)  on my own PCB, and the Display library works great !
Thanks!
![image](https://github.com/community/community/assets/24882862/0611d532-5eca-41a4-8e87-3dfbccb1bf98)

**Comment by ebndev** [comment]
Welcome to the GitHub Community, @TommieEdwardBerry, we're happy you're here! 

I've gone ahead and moved this post to our Programming Help category, as this discussion might be more relevant there! If you could provide more details about what will support the touch screen / SD support, that might help someone give you a nudge in the right direction.
